AUTHORS.txt

```diff
@@ -30,16 +31,15 @@ Former members of the AiiDA team
 * Riccardo Sabatini (2013-2015)
 
 
-Contributors
-############
-We would like to thank the following people for their major contibutions:
+## Contributors
+
+We would like to thank
 
 * Dominik Gresch (ETHZ): for the caching mechanism and contributions to the workflow engine
```

Comment:
Might as well change it to ``(Microsoft Quantum)`` if we're already updating this file.

Reply:
Hm... since you are the only two with affiliation in the file, I think it's better to remove it entirely (otherwise we have to keep updating it)

Reply:
Works for me 👍